src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -74,23 +84,49 @@ public function process(ContainerBuilder $container)
                 continue;
             }
 
+            $description = $tags[0]['description'] ?? null;
+
             unset($tags[0]);
             $lazyCommandMap[$commandName] = $id;
             $lazyCommandRefs[$id] = new TypedReference($id, $class);
-            $aliases = [];
 
             foreach ($tags as $tag) {
                 if (isset($tag['command'])) {
                     $aliases[] = $tag['command'];
                     $lazyCommandMap[$tag['command']] = $id;
                 }
+
+                $description = $description ?? $tag['description'] ?? null;
             }
 
             $definition->addMethodCall('setName', [$commandName]);
 
             if ($aliases) {
                 $definition->addMethodCall('setAliases', [$aliases]);
             }
+
+            if ($isHidden) {
+                $definition->addMethodCall('setHidden', [true]);
+            }
+
+            if (!$description) {
+                if (!$r = $container->getReflectionClass($class)) {
+                    throw new InvalidArgumentException(sprintf('Class "%s" used for service "%s" cannot be found.', $class, $id));
```

Comment:
Question: Is variable interpolation preferred over sprintf?

```
  throw new InvalidArgumentException("Class \"{$class}\" used for service \"{$id}\" cannot be found.');
```

Reply:
symfony prefers using `sprintf` for such case